MVC-DataExploration/Controllers/ProductController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace MVC_DataExploration.Controllers
+{
+    public class ProductController : Controller
+    {
+        // GET: Product
+        public ActionResult Index()
+        {
+            
+            return View();
```

Comment:
The Index view needs data from the Product table in the database. Use the database context to get this dataset and pass it to the view using 
return View(dbc.Products);